sycl/test-e2e/ESIMD/accessor_local.cpp

```diff
@@ -1,3 +1,4 @@
+// REQUIRES-INTEL-DRIVER: lin: 27202, win: 101.4677
```

Comment:
I agree that is really great feature. Thank you for adding it!

The only comment from me is is this new filter completely disables the test or only '%{run}' portion of it?

Many tests need compilation to avoid compile-time fails. Otherwise, when the driver uplift comes, we may see not the expected runtime pass, but compilation fails caused by series of changes while the test was OFF.

Reply:
Hm, thinking more about it, the argument above might not be true. However, I'd really hate to report `PASS` for something that hasn't been executed.

Reply:
Hmm, the old non-centralized checks in ESIMD monitored compile-time regressions, which was really good thing.

If only AOT compilation needs skip, then the best solution (IMO) would be to introduce %{build-aot} that is skipped if the driver doesn't meet the requirement. I have no idea though how difficult is to introduce %{build-aot}
So, if driver is too old:
  %{build} - runs
  %{build-aot} - skipped
  %{run} - skipped

Reply:
How often have you caught the problems on a test with a runtime skip that could not be observed on other tests? Was it in CI or locally? If locally, was the new/good driver available to you?

Reply:
Cannot say how often. Perhaps it is the small chance to have overlooked regression.
Compile-time regression may happen when someone else does the changes, e.g. pulls from llvm.org. CI might not catch it.

Don't consider my comments as blocking. The new centralized filter is great in general, even though it has some cons if it skips %{build}.

Reply:
If we are worried about it we could find what tests are currently skipped in CI due to old driver and create compile-only tests for them if they have APIs that aren't locked down elsewhere.

Reply:
I don't like implicit skips because the give you a false sense of all tests passing. `UNSUPPORTED` is much better in this regard. I think the right way to enable compilation is to create a fake second test that will compile-only the one disabled with this new markup:

```
# a.compile.cpp
// RUN: %{build} %S/a.cpp -o %t.out
#a.cpp
// REQUIRES-INTEL-DRIVER: ...
// NOTE: Remove a.compile.cpp once the above line is removed.
...
```

If we think `a.cpp` is important enough